app/components/meta.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Meta < ViewComponent::Base
+  renders_one :link
```

Comment:
what does this do?

Reply:
This is the way to do a slot. We insert html elsewhere using this

Reply:
nice simple syntax just its the only place ive seen it. is it confusing to have both this and the initialise block? they are both doing the same thing right?

Reply:
I don't believe so. We still pass in variables i.e. the permalink here, so that's still needed. Initialise is just a way of passing anything to the construction of a class, at least in js that's what it is :D

Reply:
i was making a case for either having both variables passed in through the initaliser, or both passed in as `renders_one`. i think i dont quite get this and prob need to have a poke tho - happy to leave for now!

Reply:
I think it would be quite a lot to pass in just a link through another slot. I think having them as this is better because the permalink is consistent, whereas the other is dynamic.

Reply:
AH ok i understand now!